common/cb_types.cwt

```diff
@@ -49,6 +49,7 @@ casus_belli = {
     ### UNKNOWN. Defaults to "no".
     is_independence = bool
 
+    ## cardinality = 0..1

```

Comment:
Update the docs for the default value then. Ditto for the rest of similar stuff.

Reply:
Aye.